### ISSUE TITLE
[HL25] Default gamma is still 2.5

### ISSUE THREAD

**Issue by qqaazz112** [ValveSoftware/halflife#3719]
The patch notes for 25th anniversary say that the default gamma has changed from 2.5 to 2.2, but it is still 2.5.

**Comment by SamVanheer** [comment]
The default value was changed, but since your config file predates the change it's set to the last value that was written to it.

**Comment by qqaazz112** [comment]
I just got the game when it was free last month. I had no prior config or save. Not a big deal, just a heads up.

**Comment by SamVanheer** [comment]
The game ships a default `config.cfg` that's installed by Steam, that might still have the old default value in it.

**Comment by qqaazz112** [comment]
Yeah that seems to be it, gamma is set to 2.5 in config.cfg on fresh install. Deleting it sets default gamma to 2.2.

**Comment by qqaazz112** [comment]
Also brightness is set to 1.0 with the default config.cfg, but is set to the minimum value of 0 when I delete it. Is 0 brightness the intended default?

**Comment by SamVanheer** [comment]
Brightness does default to 0. That may be a mistake but it's been that way since before the anniversary update.

**Comment by jakura-nuva** [comment]
> Also brightness is set to 1.0 with the default config.cfg, but is set to the minimum value of 0 when I delete it. Is 0 brightness the intended default?

I asked about this in a Steam discussion topic back when the update first launched, and the author informed me that the "brightness" slider used to be called "glare reduction". As that implies that it's only needed if glare is present, a default of "0" could make sense.

**Comment by jakura-nuva** [comment]
> I can confirm default gamma is still "2.5" on a fresh game with Steam Cloud disabled.

Coincidentally, I also did a fresh (re)install this past weekend. And yes, same results as before. The only way I've found to get brightness "0"/gamma "2.2" by default is by deleting the config, launching the game, and letting it generate a new cfg automatically. It's disappointing that new players still aren't getting the experience advertised by the anniversary update out of the box.
